make RIOT_CI_BUILD=1 CC_NOCOLOR=1 --no-print-directory -C /home/mlenders/Repositories/RIOT-OS/RIOT-release/tests/pkg_fatfs_vfs test
r
/home/mlenders/Repositories/RIOT-OS/RIOT-release/dist/tools/pyterm/pyterm -p "/dev/ttyUSB1" -b "500000" --no-reconnect --noprefix --no-repeat-command-on-empty-line
Twisted not available, please install it if you want to use pyterm's JSON capabilities
Connect to serial port /dev/ttyUSB1
Welcome to pyterm!
Type '/exit' to exit.
[auto_init_storage] error initializing sdcard_spi #0
Help: Press s to start test, r to print it is ready
READY
s
START
main(): This is RIOT! (Version: buildtest)
Tests for FatFs over VFS - test results will be printed in the format test_name:result
test_mount__mount:[FAILED]

Traceback (most recent call last):
  File "/home/mlenders/Repositories/RIOT-OS/RIOT-release/tests/pkg_fatfs_vfs/tests/01-run.py", line 34, in <module>
    sys.exit(run(testfunc))
  File "/home/mlenders/Repositories/RIOT-OS/RIOT-release/dist/pythonlibs/testrunner/__init__.py", line 30, in run
    testfunc(child)
  File "/home/mlenders/Repositories/RIOT-OS/RIOT-release/tests/pkg_fatfs_vfs/tests/01-run.py", line 28, in testfunc
    raise TestFailed(child.after.split(':', 1)[0] + " test failed!")
__main__.TestFailed: test_mount__mount test failed!
make: *** [/home/mlenders/Repositories/RIOT-OS/RIOT-release/tests/pkg_fatfs_vfs/../../Makefile.include:770: test] Error 1

Return value: 2
